manager/system_handler.py

```diff
@@ -206,7 +207,7 @@ def _full_query(rh_account_id, query_args, parsed_args, filters, remediation_fil
                         fn.COALESCE(SystemVulnerabilities.remediation_type_id, remediation.PLAYBOOK.value).alias("remediation_type_id"),
                         )
                 .join(SystemPlatform, on=((SystemVulnerabilities.system_id == SystemPlatform.id) &
-                                          system_is_active(edge=None, stale=None, rh_account_id=rh_account_id)))
+                                          system_is_active(edge=edge_feature_arg(), stale=None, rh_account_id=rh_account_id)))
```

Comment:
CVEs for an edge system should always be returned, regardless of the feature flag. `edge=None` should be correct.

Reply:
You are right